src/utils/overwriteProfileDefaults.js

```diff
@@ -0,0 +1,8 @@
+export function overwriteProfileDefaults({ email, name, avatar }) {
+  const [emailName] = email.split('@');
+
+  return {
+    name: emailName === name ? '' : name,
```

Comment:
The original requirements stated that we should not populate the fields if they were the default ones from auth0, but in another comment it was mentioned that this is not necessary as long as the validation requires a space in the name string.

I went ahead and removed this value anyway, but if we want to stick to the validation only, I can remove this part :)

Reply:
The space validation will block that case too so it will never gonna happened. Right?

Reply:
Yes that is correct, I just thought it might be better for user experience since the error message for invalid fields is not very descriptive at the moment.

Reply:
Good point. We can keep it for now. If we'll see a need to change it, will do.